docs/content/guides/getting-started/events-and-hooks/events-and-hooks.md

```diff
@@ -156,7 +156,8 @@ It's worth mentioning that some Handsontable hooks are triggered from the Handso
 | -------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `auto`                                             | Action triggered by Handsontable, and the reason for it is related directly to the settings applied to Handsontable. For example, [`afterCreateRow`](@/api/hooks.md#aftercreaterow) will be fired with this when [`minSpareRows`](@/api/options.md#minsparerows) will be greater than 0. |
 | `edit`                                             | Action triggered by Handsontable after the data has been changed, e.g., after an edit or using `setData*` methods.                                                                                                     |
-| `loadData`                                         | Action triggered by Handsontable after the [`loadData`](@/api/core.md#loaddata) or [`updateSettings({data: myData})`](@/api/core.md#updatesettings) with the [`data`](@/api/options.md#data) property method has been called.                                                                             |
+| `loadData`                                         | Action triggered by Handsontable after the [`loadData`](@/api/core.md#loaddata) with the [`data`](@/api/options.md#data) property method has been called.
```

Comment:
```suggestion
| `loadData`                                         | Action triggered by Handsontable after the [`loadData`](@/api/core.md#loaddata) method has been called with the [`data`](@/api/options.md#data) property.
```
@AMBudnik would you think this is more correct?

Reply:
@evanSe yea, that `loadData()` looks good now